### PR TITLE
docs: update terminology

### PR DESCRIPTION
**Pull request by danieltprice**
This PR updates storage and read replica terminology

### CODE REVIEW

File: content/docs/get-started-with-neon/production-readiness.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ subtitle: Neon features for real-world workloads
 enableTableOfContents: true
 ---
 
-Learn how autoscaling, autosuspend, bottomless storage, change data capture, and instant-read replicas can improve performance, reliability, and efficiency for your production environments.
+Learn how autoscaling, scale-to-zero, Neon's purpose-built storage architecture, change data capture, read replicas, and support for thousands of connections can improve performance, reliability, and efficiency for your production environments.
```
Comment (bgrenon) [inline content/docs/get-started-with-neon/production-readiness.md:7]: "purpose-built" feels a bit vague. What about "Neon's separated storage architecture" or "Neon's disaggregated storage architecture"?

Comment (bgrenon) [reply content/docs/get-started-with-neon/production-readiness.md:7]: Not sure either of those is better, but I did pause on purpose-built. "cloud-native" is maybe another option

File: content/docs/get-started-with-neon/production-readiness.md [hunk 2]
```diff
@@ -31,11 +31,11 @@ Neon's _Autosuspend_ feature automatically transitions a Neon compute instance (
...
 Learn more about [why you want a database that scales to zero](https://neon.tech/blog/why-you-want-a-database-that-scales-to-zero).
 
-## Bottomless storage
+## A storage architecture purpose-built for the cloud
 
-**Any data size; any growth trajectory.**
+**Efficient, performant, reliable storage**
 
-Neon's storage system was purpose-built for the cloud to provide virtually unlimited storage with high availability and durability guarantees. Neon stores every transaction in multiple copies across availability zones and S3. Efficiency and performance are achieved through a multi-tier architecture designed to balance latency, throughput, and cost considerations.
+Neon's storage system was purpose-built for the cloud to provide storage with high availability and durability guarantees. Neon stores every transaction in multiple copies across availability zones and S3. Efficiency and performance are achieved through a multi-tier architecture designed to balance latency, throughput, and cost considerations.
```
Comment (bgrenon) [inline content/docs/get-started-with-neon/production-readiness.md:38]: purpose-built seems fine here, as we state the purpose: for the cloud

**Review by bgrenon — APPROVED** [review]
Minor comment about "purpose-built" you can disregard if that terms' been shopped around already.

Really like this one: "A serverless Postgres platform designed to help developers build reliable and scalable applications faster."

File: content/docs/get-started-with-neon/production-readiness.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/get-started-with-neon/production-readiness.md]: Thank you. Dropped "purpose-built"

**Pull request merged**